src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -50,6 +50,21 @@ internal bool AutomaticRedirection
             }
         }
 
+        internal ClientCertificateOption ClientCertificateOptions
+        {
+            get
+            {
+                return ClientCertificateOption.Manual;
+            }
+            set
+            {
+                if (ClientCertificateOption.Manual != value)
+                {
+                    throw PlatformNotSupportedException("libcurl does not support automatic client certificate selection");
```

Comment:
This won't compile; it's missing ```new```. And the message should be in the resources so that it can be localized.